### ISSUE TITLE
Duplicate class/interface generated

### ISSUE THREAD

**Issue by clemens-tolboom** [clue/graph-uml#15]
When generating diagram with

```
\Fhaculty\Graph\Edge\Base
\Fhaculty\Graph\Edge\Directed
```

adds `Fhaculty\Graph\Edge\Base` twice.

![duplicate-fhaculty graph edge base](https://f.cloud.github.com/assets/371014/1819461/8b95bc5a-7095-11e3-987f-ce629f84cadc.png)

**Comment by clue** [comment]
Indeed, thanks for reporting! :+1: 

Looks like we do not remove the leading namespace separator (`\`) for the internal vertex ID. The following code should fix this:

``` php
$classname = ltrim($classname, '\\');
```

I'll add this to my (lengthy) todo list, so if you happen to find the time to check this first, let me know :)

**Comment by clemens-tolboom** [comment]
Fixed in PR #24
